src/App.js

```diff
@@ -9,19 +9,24 @@ import NavBar from './Components/NavBar';
 import ErrorModal from './Components/ErrorModal'
 import { NavigationProvider } from './Context/NavigationContext';
 import { ErrorProvider } from './Context/ErrorContext'
+import { CryptidProvider } from './Context/CryptidContext';
+import SingleCryptidSightings from './Components/SingleCryptidSightings';
 
 const App = () => {
   return (
     <ErrorProvider>
       <NavigationProvider>
-        <NavBar />
-        <ErrorModal />
-        <Routes>
-          <Route path="/" element={<Home />} />
-          <Route path="/singlecryptid/:name" element={<SingleCryptid />} />
-          <Route path="/allsightings" element={<AllSightings />} />
-          <Route path="/singlesighting/:name" element={<SingleSighting />}/>
-        </Routes>
+        <CryptidProvider>
+          <NavBar />
+          <ErrorModal />
+          <Routes>
+            <Route path="/" element={<Home />} />
+            <Route path="/singlecryptid/:name" element={<SingleCryptid />} />
+            <Route path="/allsightings" element={<AllSightings />} />
+            <Route path="/singlesighting/:name" element={<SingleSighting />} />
+            <Route path="/singlecryptidsightings/:name" element={<SingleCryptidSightings />} />
```

Comment:
Smart to have a new component for single cryptids

Reply:
Great idea to be able to use the link for routing.